packages/test/src/generated/shop-graphql.ts

```diff
@@ -3424,6 +3424,7 @@ export const OrderFields = gql`
       type
     }
   }
+  couponCodes
```

Comment:
Please make sure you edit the .graphql files and not the generated files liek this one. This will get overwritten when types are regenerateed.

Te correct flow is:

1. Add fields in `test/src/shop.graphql`
2. Start the dev server with `yarn start` inside `/packages/test`folder
3. In another terminal window, run `yarn generate`, also in the `/test` folder.

Reply:
okay